src/pages/settings/InitialSettingsPage.js

```diff
@@ -326,79 +327,80 @@ function InitialSettingsPage(props) {
     if (_.isEmpty(props.currentUserPersonalDetails)) {
         return null;
     }
-
-    return (
-        <ScreenWrapper includeSafeAreaPaddingBottom={false}>
-            {({safeAreaPaddingBottomStyle}) => (
+    const headerContent = (
+        <View style={[styles.avatarSectionWrapper, styles.justifyContentCenter]}>
+            {_.isEmpty(props.currentUserPersonalDetails) || _.isUndefined(props.currentUserPersonalDetails.displayName) ? (
+                <CurrentUserPersonalDetailsSkeletonView
+                    backgroundColor={themeColors.appBG}
```

Comment:
Avatar loading skeleton color was same as what we use for Header of the layout so have to change the color to same a body color. @shawnborton can you review this change also.

| If we dont change color to appBG | Now |
|--------|--------|
| <img width="1822" alt="Screenshot 2023-09-20 at 3 13 34 PM" src="https://github.com/Expensify/App/assets/104348397/55698ca8-5f1d-4493-89a4-b344a7ad128c"> | <img width="1822" alt="Screenshot 2023-09-20 at 3 14 03 PM" src="https://github.com/Expensify/App/assets/104348397/94ddb5b7-6b78-49b5-b490-fbb0a4633016"> |